### DISCUSSION TITLE
How to view/set the presets via MQTT?

### DISCUSSION THREAD

**Discussion by forTT78** [omerfaruk-aran/esphome_samsung_hvac_bus#148] [Questions]
Hi together,
i would like to ask if can see and change some presets (like beep, display, windfree)  via MQTT, is there a way?
E.g. activate/deactivate windfree?

Via MQTT I see for eg. the swing_mode but not if windfree is active or not.

May be someone has a hint for me.

best regards,
Thomas

**Comment by Oggy123** [comment]
Hi Thomas, yes, it's possible. You can control them in topic climate/unit_name/preset/command, but first you have enable it in yaml section capabilities -> presets

**Reply by forTT78** [reply]
Nice! Thank you. I will try it.
Is there a list of all available presets?

**Reply by Oggy123** [reply]
From example.yaml:
# The following modes are available: sleep, quiet, fast, longreach, windfree

**Reply by forTT78** [reply]
Hmm, ok. I was looking for something what I use on the remote under options:
like "beep off", "display off" - is these kind of setting also a preset?

**Comment by Oggy123** [comment]
It's been discussed in another topic, these options are probably not published on the bus. AFAIK even with original modbus card you can't modify these options. For me is best combination this project (for reading current states) and ESPEasy with IRremoteESP8266 (for sending commands, because it's possible control beep and display).
